status/projects.py

```diff
@@ -693,6 +695,34 @@ def notify_tagged_user(application, userTags, project, note, category, tagger, t
                 s.sendmail('genomics-bioinfo@scilifelab.se', msg['To'], msg.as_string())
                 s.quit()
 
+                #Adding a slack IM to the tagged user with the running note
+                nest_asyncio.apply()
+                client = slack.WebClient(token=application.slack_token)
+
+                blocks = [
+                            {
+                              "type": "section",
+		                      "text": {
+                                "type": "mrkdwn",
+        		                "text": ("_You have been tagged by *{}* in a running note for the project_ "
+                                       "<{}/project/{}|{}>! :smile: \n_The note is as follows:_ \n\n\n")
+                                        .format(tagger, application.settings['redirect_uri'].rsplit('/',1)[0], project, project) 
+                               }
+                            },
+                            {
+                              "type": "section",
+                              "text": {
+                                "type": "mrkdwn",
+                                "text": ">*{} - {}{}*\n>{}\n\n\n\n _(Please do not respond to this message here in Slack."
+                                      " It will only be seen by you.)_".format(tagger, time_in_format, category, note.replace('\n', '\n>'))
+         	                   }
+                            }
+                         ]
```

Comment:
Still looks a bit wonky 🤔 Maybe try this?
```
                blocks = [
                    {
                        "type": "section",
                        "text": {
                            "type": "mrkdwn",
                            "text": ("_You have been tagged by *{}* in a running note for the project_ "
                                     "<{}/project/{}|{}>! :smile: \n_The note is as follows:_ \n\n\n")
                            .format(tagger, application.settings['redirect_uri'].rsplit('/',1)[0], project, project)
                        }
                    },
                    {
                        "type": "section",
                        "text": {
                            "type": "mrkdwn",
                            "text": ">*{} - {}{}*\n>{}\n\n\n\n _(Please do not respond to this message here in Slack."
                            " It will only be seen by you.)_".format(tagger, time_in_format, category, note.replace('\n', '\n>'))
                        }
                    }
                ]
```